.github/workflows/build-all-matrix.yaml

```diff
@@ -226,7 +234,9 @@ jobs:
         uses: jmgilman/actions-generate-checksum@v1
         with:
           method: sha512
-          patterns: "*.tar.gz"
+          patterns: |
+            "*.tar.gz"
+            "*.iso"
```

Comment:
@jacobweinstock I think you've unwanted quotes here; this will cause checksum to be empty, I think.

Reply:
Ah thank you! i've been looking for why the checksum wasnt being created!